lib/internal/modules/cjs/loader.js

```diff
@@ -122,9 +122,7 @@ Module._extensions = Object.create(null);
 var modulePaths = [];
 Module.globalPaths = [];
 
-Module.wrap = function(script) {
-  return Module.wrapper[0] + script + Module.wrapper[1];
-};
+Module.wrap = (script) => (Module.wrapper[0] + script + Module.wrapper[1]);
```

Comment:
Again, there is no need for the parenthesis here. Also as long as this is being changed, perhaps it could become a template literal?